client/src/components/Login/Login.tsx

```diff
@@ -5,7 +5,7 @@ import {
 
 function Login() {
   const connectSpotify = () => {
-    window.location.href = 'http://localhost:9000/auth/login'; // is there a way to thunk this?
+    window.location.href = 'http://localhost:9000/auth/login';
```

Comment:
For now, store the url base ('http://localhost:9000/') as an environment variable so that it can easily be substituted upon deployment?

Reply:
ooo good call

Reply:
okay, fixed